www/include/monitoring/downtime/xml/broker/makeXMLForDowntime.php

```diff
@@ -120,21 +120,23 @@
 }
 
 $rowClass = "list_one";
-while ($row = $stmt->fetch()) {
-    $row['comment_data'] = strip_tags($row['comment_data']);
-    $xml->startElement('dwt');
-    $xml->writeAttribute('class', $rowClass);
-    $xml->writeElement('author', $row['author']);
-    $xml->writeElement('start', $row['actual_start_time']);
-    if (!$row['fixed']) {
-        $row['end_time'] = (int)$row['actual_start_time'] + (int)$row['duration'];
+if (isset($res)) {
```

Comment:
why do you check $res ? It should always exist

Reply:
Same expectation than kduret , 
when initiate $res should be a PDOStatement instance if prepare() method went well, or false if something wrong happened. Then it should be true or false as result of execute() method. 
So in every case $res is set but you should test if $res === true

Reply:
I forgot to remove this line from debug. This line isn't required anymore since I added the host_id check on line 73 in the last PR ;)
@jeremyjaouen -> before the modifications, if host_id was not set, the PDO failed when binding false on host_id in the query of the second case